test/release/examples/primers/modules.chpl

```diff
@@ -558,15 +566,24 @@ module MainModule {
       // writeln(bar);        // this won't resolve since bar isn't available
     }
 
-    /* When any of these are present, any instances of classes or records
-       will be able to access their symbols defined in that module.
+    /* Class and record instances obtained in scopes where the type is not
+       otherwise visible can still access any fields and any methods defined in
+       their original scope.
```

Comment:
"the type" --> "their type"

"their original scope" --> "their type's original scope"